cluster-cm/src/main/java/com/sequenceiq/cloudbreak/cm/ClouderaManagerSetupService.java

```diff
@@ -270,12 +271,15 @@ public void installCluster(String template) {
                             .importClusterTemplate(calculateAddRepositories(apiClusterTemplate, prewarmed), apiClusterTemplate);
                     ClusterCommand clusterCommand = new ClusterCommand();
                     clusterCommand.setClusterId(cluster.getId());
-                    clusterCommand.setCommandId(apiCommand.getId());
+                    clusterCommand.setCommandId(BigDecimal.valueOf(apiCommand.getId()));
```

Comment:
Does this still need `BigDecimal` then?

Reply:
unfortunately the CM api is not as consistent than you think :D